.github/workflows/main.yml

```diff
@@ -357,24 +357,24 @@ jobs:
       shell: bash
     - name: Check Docfx cache
       id: check-docfx-cache
-      if: ${{ contains(github.ref, 'release') }}
+      if: ${{ contains(github.head.ref, 'release') }}
       uses: actions/cache@v2
       with:
         path: C:\docfx
         key: docfx
     - name: Download docfx
-      if: ${{ steps.check-docfx-cache.outputs.cache-hit != 'true' && contains(github.ref, 'release') }}
+      if: ${{ steps.check-docfx-cache.outputs.cache-hit != 'true' && contains(github.head.ref, 'release') }}
```

Comment:
@DominicFrei good catch. I didn't notice because of the structure of the templated `main.yml`. I rearranged the template code. How does it look (line 106-107, 116 in `templates/main.yml`)?

Reply:
Look good to me. Would you then want to use this substitution for `contains(github.head.ref, 'release')` in line 366 as well?

Reply:
I'm not sure I follow. It's done in line 366. What do you mean?

Reply:
No worries.

Here is what I meant:
Line 106 was added:
```#@ isRelease = "contains(github.head.ref, 'release')"```

It was then used in line 116:
```    if: #@ "${{ steps.check-docfx-cache.outputs.cache-hit != 'true' && " + isRelease + " }}"```

The same could be done in line 366:
```      if: ${{ steps.check-docfx-cache.outputs.cache-hit != 'true' && contains(github.head.ref, 'release') }}```
could be
```      if: ${{ steps.check-docfx-cache.outputs.cache-hit != 'true' && isRelease }}```

The `contains(github.head.ref, 'release')` could also be substituted with `isRelease` since you created it for that purpose, right?

In fact, you could also use it in line 360, 372 and 377:
```      if: ${{ contains(github.head.ref, 'release') }}```
could be 
```      if: ${{ isRelease }}```

Reply:
```suggestion
      if: ${{ steps.check-docfx-cache.outputs.cache-hit != 'true' && isRelease }}
```

Reply:
I've added them all as change suggestions. Let me know if that makes sense. :)